### PR TITLE
Adding versioning to run with 275 (arches) or 41 (brycecanyon) GeV beam with B0 ECal

### PR DESCRIPTION
**Pull request by rahmans1**
Adding versioning to run with 275 (arches) or 41 (brycecanyon) GeV beam with B0 ECal

- [x] This should be merged following https://github.com/eic/ip6/pull/45

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [X] New feature (issue #160)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [X] Tests for the changes have been added
- [X] Documentation has been added / updated
- [X] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
No
### Does this PR change default behavior?
Introduces versioning in ip6 geometry (arches uses ecal in B0 and brycecanyon uses preshower)

### CODE REVIEW

**Comment by wdconinc** [comment]
Any idea why the lcg view job is failing? Just due to the ip6 PR not being merged?

Is this otherwise ready? If so, can we remove "Draft" from the title?

**Comment by rahmans1** [comment]
> Any idea why the lcg view job is failing? Just due to the ip6 PR not being merged?
> 
> Is this otherwise ready? If so, can we remove "Draft" from the title?

I think it's ready. It seemed to pass all the tests here https://github.com/eic/epic/actions/runs/3207491282/jobs/5245490398.

File: configurations/brycecanyon.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+features:
+  beampipe:
+  tracking:
+  pid:
+    tof_endcap:
+    tof_barrel:
+    mrich:
+    drich:
+      #SJJ: disabled as it needs re-integrating with the tracker
+      #    tof_barrel:
```
Comment (wdconinc) [inline configurations/brycecanyon.yml:10]: Remove unnecessary comment lines.
```suggestion
```

**Review by wdconinc — DISMISSED** [review]

**Review by veprbl — APPROVED** [review]

**Review by veprbl — DISMISSED** [review]

**Comment by rahmans1** [comment]
@DraTeots @wdconinc All benchmarks now pass. This should be ready to merge following https://github.com/eic/ip6/pull/45

File: configurations/arches.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+features:
```
Comment (wdconinc) [inline configurations/arches.yml:1]:
```suggestion
ebeam: 18
pbeam: 275
features:
```

**Review by wdconinc — APPROVED** [review]
Looks good to me.

**Pull request merged**